### PR TITLE
added cff file for Zenodo publishing

### PR DESCRIPTION
**Pull request by JFRudzinski**
@JosePizarro3 I have been instructed to start publishing our major software on Zenodo. I already activated this repo, so it will be published and a DOI generated at the next release. We just need to finalize the cff file. Let me know if you have any suggested changes.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,32 @@
+cff-version: 1.2.0
+title: NOMAD Simulations Schema Plugin
```
Comment (JosePizarro3) [inline CITATION.cff]: Only "NOMAD Simulations"

I prefer to separate concepts like schema or plugin from the repo itself (although it works being compliant with it)

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  metadata from this file.
+type: software
+authors:
+  - given-names: Nathan
+    family-names: Daelman
+    orcid: 'https://orcid.org/0000-0002-7647-1816'
+  - given-names: Bernadette
+    family-names: Mohr
+    orcid: 'https://orcid.org/0000-0003-0903-0073'
+  - given-names: José M.
```
Comment (JosePizarro3) [inline CITATION.cff]: I'd like to be main author and you (@JFRudzinski) at the end

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    family-names: Pizarro
+    orcid: 'https://orcid.org/0000-0002-6751-8192'
+  - given-names: Esma B.
+    family-names: Boydas
+    orcid: ''
```
Comment (JosePizarro3) [inline CITATION.cff]: @EBB2675 do you have orcid? I am sure Joe asked you already

File: CITATION.cff [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  - given-names: Joseph F.
+    family-names: Rudzinski
+    orcid: 'https://orcid.org/0000-0003-3403-640X'
+doi:
```
Comment (JosePizarro3) [inline CITATION.cff:26]: i guess here you need to add the zenodo doi?

File: CITATION.cff [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+repository-code: 'https://github.com/nomad-coe/nomad-simulations'
+url: 'https://nomad-coe.github.io/nomad-simulations/'
+abstract: >-
+  `nomad-simulations` is a plugin for the NOMAD software (https://zenodo.org/doi/10.5281/zenodo.8366162)
```
Comment (JosePizarro3) [inline CITATION.cff:30]: """
`nomad-simulations` is a plugin for the NOMAD software (https://zenodo.org/doi/10.5281/zenodo.8366162)
providing a common data standard and analysis tools for parsed Materials Science and Quantum Chemistry
simulations. This standard is used as a template populated by the many available parsers to transform their
inputs / outputs into a structured data format. This plugin is community-driven, and users are encourage to
contribute towards providing Findable, Accessible, Interoperable, and Reusable (FAIR) data.
"""

Let me know what you think. I wanted to put a bit more the emphases on the analysis that the normalizers provide, and not only on the RDM aspect.

**Review by JosePizarro3 — APPROVED** [review]
Great!

I saw the nomad-material-processing, and they have a badge from Zenodo. Can you also add this or you need to first publish this, and then we modify the README?

File: CITATION.cff (hunk 2 above)
Comment (JFRudzinski) [reply CITATION.cff]: yeah, I was going to do that at first. Ok, then I leave the others in alphabetical order in the middle?

File: CITATION.cff (hunk 3 above)
Comment (JFRudzinski) [reply CITATION.cff]: I added it now

File: CITATION.cff (hunk 4 above)
Comment (JFRudzinski) [reply CITATION.cff:26]: yeah, but it seems a bit cyclic, I need to make a release to get the DOI, and then add it here after which will then be present in the next release :shrug:

File: CITATION.cff (hunk 5 above)
Comment (JFRudzinski) [reply CITATION.cff:30]: I used yours with just a very small adjustment, you can check.

File: CITATION.cff (hunk 2 above)
Comment (JosePizarro3) [reply CITATION.cff]: I think it makes sense to add them in order of contribution.

would you mind adding Alvin as well?

File: CITATION.cff (hunk 4 above)
Comment (JosePizarro3) [reply CITATION.cff:26]: Ok, no problem. Then good as it is, then later we change doi and README with the badge

File: CITATION.cff (hunk 5 above)
Comment (JosePizarro3) [reply CITATION.cff:30]: Sure, lgtm 👍🏻

Comment (JosePizarro3) [reply CITATION.cff:30]: sure lgtm!

File: CITATION.cff (hunk 4 above)
Comment (JFRudzinski) [reply CITATION.cff:26]: I think I found a way to add the badge before the first release, I followed this: https://gist.github.com/seignovert/ae6771f400ca464d294261f42900823a

Comment (JosePizarro3) [reply CITATION.cff:26]: Ok cool. Would you mind adding at the begining with the other badges? 

We could also add a citation before the contacts table for `## How to cite this work`

File: CITATION.cff (hunk 2 above)
Comment (JFRudzinski) [reply CITATION.cff]: @ladinesa FYI - I am adding you as author in our Zenodo publication

File: README.md [hunk 6]
```diff
@@ -160,6 +161,9 @@ plugins:
 **Note!**
 Once you modify your `nomad.yaml` file adding `include`, all the default plugins will be disconnected, so you will need to include them as well.
 
+## How to cite this work
```
Comment (JFRudzinski) [inline README.md:164]: @JosePizarro3 like this?

Comment (JosePizarro3) [reply README.md:164]: Pizarro, J.M., Boydas, E.B., Daelman, N., Ladines, A.N., Mohr, B. & Rudzinski, J.F., NOMAD Simulations [Computer software]. https://doi.org/xxxxx

Comment (JosePizarro3) [reply README.md:164]: tho I think a new option will open on the menu on the right for citation, if i see nomad-material-processing

Comment (JFRudzinski) [reply README.md:164]: ah ok, thanks. Well, it does't hurt to have it in the README, it's a bit more visible...merge?

**Comment by JosePizarro3** [comment]
merge at will

**Pull request merged**